spec/system/dashboard_system_spec.rb

```diff
@@ -162,8 +162,9 @@
         ["Last 30 Days", test_time - 29.days,                     test_time],
         ["This Month",   test_time.beginning_of_month,            test_time.end_of_month],
         ["Last Month",   test_time.last_month.beginning_of_month, test_time.last_month.end_of_month],
-        ["This Year",    test_time.beginning_of_year,             test_time.end_of_year],
-        ["All Time",     test_time - 100.years,                   test_time]
+        ["This Year",    test_time.beginning_of_year,             test_time.end_of_year]
+        #  We now can't test the lower limit of All Time, because the earliest possible date is 2000-01-01
+        #  ["All Time",     test_time - 100.years,                   test_time]
```

Comment:
Sorry - shouldn't we still have "All Time" be "since 2000"?

Reply:
Pretty sure the same thing is going to apply here -- that the tests will try to create things that can't be created in order to check that All Time excludes them.

Reply:
Yup.  confirmed that.

Reply:
We don't have to check that All Time _excludes_ something - at the very least we can test that it _includes_ something that the other values don't.

Reply:
I've just been avoiding tinkering with the "do this for all the preset ranges" code, but we could do a single test for all time outside of that, I suppose.

Reply:
It turned out to be easier to remove testing one end of "All Time" than to add in the 5 or 6 extra tests that would have been needed.